content/hardware/tplink.md

```diff
@@ -12,53 +12,55 @@ An indoor router, connected by ethernet cable to the outdoor router, is required
 
 
 ### Pre-Configuration Best Practices
-**for volunteer installers**
+**For volunteer installers**
 
 Before configuring, ask the user member to tell you:
+
 * Whether they would like to use the TP-Link router we supply or their own router
 * A username and password for the router
 * An SSID and password for the home wireless network
 
 ### TP-Link TL-WR841N Router Configuration Instructions
-**for volunteer installers and DIY installers**
+**For volunteer installers and DIY installers**
+
+![alt text](https://i.imgur.com/5BVxd9h.jpg "TP-Link Router")
```

Comment:
Moved image to top of section to fix the numbered list formatting issue that was occuring due to list breaks. (essentially, this change and others made to this section will fix the list so its numbered from 1 - 8 as intended, and not from 1-2, 1-3, 1-4 etc at every list break. This is a bug with Markdown IMO